client.js

```diff
@@ -1,14 +1,33 @@
-const Orbs = require("orbs-client-sdk");
+const { Client, NetworkType } = require("orbs-client-sdk");
+const NotaryContractName = process.env.NOTARY_CONTRACT_NAME;
+const ORBS_TOPOLOGY = JSON.parse(process.env.ORBS_TOPOLOGY);
+const VCHAIN = Number(process.env.VCHAIN);
 
-const NotaryContractName = process.env.NOTARY_CONTRACT_NAME || "Notary";
-const API_ENDPOINT = process.env.API_ENDPOINT || "http://localhost:8080";
-const VCHAIN = Number(process.env.VCHAIN || 42);
+function buildClient(url) {
+    return new Client(url, VCHAIN, NetworkType.NETWORK_TYPE_TEST_NET);
+}
+
+function buildUrl(node) {
+    const { host } = node;
+    const prefix = `http://${host}`;
+    if (host.includes("localhost")) {
+        return prefix;
+    } else {
+        return `${prefix}/vchains/${VCHAIN}`;
+    }
+}
 
-function getClient() {
-    return new Orbs.Client(API_ENDPOINT, VCHAIN, Orbs.NetworkType.NETWORK_TYPE_TEST_NET);
+function getClients() {
+    if (!ORBS_TOPOLOGY || !ORBS_TOPOLOGY.nodes) {
+        return [];
+    }
+    const { nodes } = ORBS_TOPOLOGY;
+    return nodes
+        .map(buildUrl)
+        .map(buildClient);
```

Comment:
I would like to get a sample of clients (random 1/3 out of all, number is configurable) not smaller than 2 (minimal size of the network is 4).

It should be sufficient in case some nodes go down but the network as a whole is still alive.